test/evm_test.exs

```diff
@@ -168,22 +189,31 @@ defmodule EvmTest do
   end
 
   def passing_tests(test_group_name) do
-    passing_tests_in_group =
+    tests =
       if Map.get(@passing_tests_by_group, test_group_name) == :all do
         all_tests_of_type(test_group_name)
       else
         Map.get(@passing_tests_by_group, test_group_name)
       end
-      |> Enum.map(fn test_name ->
-        {test_name, read_test_file(test_group_name, test_name)}
-      end)
+
+    Enum.map(tests, fn test_data ->
```

Comment:
do we need the variable at all? ie:

    if Map.get(@passing_tests_by_group, test_group_name) == :all do
     all_tests_of_type(test_group_name)
    else
      Map.get(@passing_tests_by_group, test_group_name)
    end |>
      Enum.map(fn test_data ->
       ...

Reply:
This version caused credo warning `Pipe chain should start with a raw value`

Reply:
so it's ok?

Reply:
ah, yep 👍